README.md

```diff
@@ -152,24 +127,15 @@ Using angle bracket invocation:
 
 Two arguments are passed to the block: `message` object, and `close` action. Make sure
 you are using *Closure Actions* syntax passing the action (e. g. `<a {{action close}}` or
-`{{your-component close=(action close)}}`.
+`<YourComponent @close={{action close}} />`.
```

Comment:
`this` binding should be done internally using `@action` decorator to drop that requirement and simplify to `<YourComponent @close={{close}} />`. But will leave that one for another PR.